src/Orchard.Web/Modules/Orchard.Autoroute/ImportExport/HomeAliasImportHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Linq;
+using System.Web.Routing;
+using Orchard.Autoroute.Services;
+using Orchard.ContentManagement;
+using Orchard.Recipes.Models;
+using Orchard.Recipes.Services;
+
+namespace Orchard.Autoroute.ImportExport {
+    public class HomeAliasImportHandler : Component, IRecipeHandler {
+        private readonly IHomeAliasService _homeAliasService;
+        private readonly IContentManager _contentManager;
+
+        public HomeAliasImportHandler(IHomeAliasService homeAliasService, IContentManager contentManager) {
+            _homeAliasService = homeAliasService;
+            _contentManager = contentManager;
+        }
+
+        public void ExecuteRecipeStep(RecipeContext recipeContext) {
+            if (!String.Equals(recipeContext.RecipeStep.Name, "HomeAlias", StringComparison.OrdinalIgnoreCase)) {
```

Comment:
Lowercase `string` when specifying a variable type, yes. But not when referring to the `String` class, as is the case here. The following code is correct IMO:

    string x = String.Format("{0}", number);

The first is typing a variable, using a type keyword. The second is referring to a static method on a class. It's very weird IMO to use lowercase in the latter case - you're not calling a static method on a type keyword.